sdk/core/core-http/src/policies/disableResponseDecompressionPolicy.browser.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/*
+ * NOTE: When moving this file, please update "browser" section in package.json
+ */
+import { BaseRequestPolicy, RequestPolicy, RequestPolicyOptions } from './requestPolicy';
+import { WebResource } from '../webResource';
+import { HttpOperationResponse } from '../httpOperationResponse';
+
+const DisbleResponseDecompressionNotSupportedInBrowser = new Error("DisableResponseDecompressionPolicy is not supported in browser environment");
+
+export interface CompressOptions {
+  enable: boolean;
+}
+
+export function disableResponseDecompressionPolicy(_compressOptions?: CompressOptions) {
```

Comment:
Does this need compressOptions? Doesn't look like it is used.

Reply:
No, I forgot to update the browser file.

Reply:
Updated